README.md

```diff
@@ -1,27 +1,37 @@
-# Project Name
+# FCC PENGUIN
 
-> One paragraph statement about the project.
+> A penguin project built using animation, transform to rotate elements and scale them.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./_C__projects_fcc_fcc-penguin_index.html.png)
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML5 elements
+  - `div`.
+- CSS properties
+  - CSS transform such as `rotate`,
```

Comment:
Tis is fine but what do you think of ......CSS Typography `transform` `transform-origin`???

Nice code 👍